src/libsync/propagateremotemkdir.h

```diff
@@ -60,5 +57,8 @@ private slots:
     void propfindResult(const QVariantMap &);
     void propfindError();
     void success();
+
+private:
+    bool _isEncryptedRootFolder = false;
```

Comment:
I don't like the fact that the other members are at the top of the class like this but I like even less the fact that now we got them at two places... so I'd say either you move _isEncryptedRootFolder with the other ones, or you move the other ones down in that private section (it's not strictly related to this change but that's a good opportunity to "fix" this and it's still easily intelligible).